src/synopsys-action/diagnostics.ts

```diff
@@ -0,0 +1,33 @@
+import * as artifact from '@actions/artifact'
+import {UploadResponse} from '@actions/artifact/lib/internal/upload-response'
+import {getWorkSpaceDirectory} from '@actions/artifact/lib/internal/config-variables'
+import * as fs from 'fs'
+import * as inputs from './inputs'
+import {UploadOptions} from '@actions/artifact/lib/internal/upload-options'
+
+export async function uploadDiagnostics(): Promise<UploadResponse> {
+  const artifactClient = artifact.create()
+  const pwd = getWorkSpaceDirectory().concat('/.bridge/')
```

Comment:
If we change the default bridge download path, means if bridge is downloaded separate to project directory then this code needs to be modified?

Reply:
bridge download path nothing to do with diagnostics file, as these files will be stored under current workspace followed by .bridge folder